src/components/CalendarComponents/categoriesBar.vue

```diff
@@ -37,32 +38,39 @@ import { AllCategories } from '@/db-service/Categories/queries';
   },
 })
 export default class CategoriesBar extends Vue {
-  get categories(): Category[] {
-    if (this.$data.allCategories !== undefined) {
-      return this.$data.allCategories.nodes;
+    @Prop()
+    title!: string;
+
+    get categories(): Category[] {
+      if (this.$data.allCategories !== undefined) {
+        return this.$data.allCategories.nodes;
+      }
+      return [];
     }
-    return [];
-  }
 }
 </script>
 
 <style scoped>
+    .nav-button {
+        margin: 0 0.25vw;
+        flex: 1;
+    }
     .add-button {
         border: solid black 1px;
-        height: 5vh;
-        width: 5vh;
+        flex: 0.75;
     }
     .categories {
         display: flex;
         justify-content: center;
         align-items: center;
         width: 100%;
+        flex: 25;
     }
     .category {
         font-weight: bold;
         border-radius: 9%;
         border: solid black 0.5px;
-        width: 8%;
```

Comment:
האורך הזה הגיע מתיאום עם רוני

Reply:
This size isn't enough for the meeting category